lightkurve/targetpixelfile.py

```diff
@@ -1130,6 +1131,163 @@ def cutout(self, center=None, size=5):
+            for i in range(self.shape[1]*self.shape[2]):
+                masks[i][np.unravel_index(i, (self.shape[1], self.shape[2]))] = True
+            
+            pixel_list = []
+
+            for j in range(self.shape[1]*self.shape[2]):
+
+                lc = self.to_lightcurve(aperture_mask=masks[j])
+
+                if normalize: # overrides corrector function
+                    corrector_func = lambda x:x.normalize().remove_outliers()
+                    lc_corr = corrector_func(lc)
+                elif corrector_func == None:
+                    corrector_func = lambda x:x.normalize().remove_outliers()
+                    lc_corr = corrector_func(lc)
+                else:
+                    lc_corr = corrector_func(lc)
+
+                if periodogram:
+                    try:
+                        pixel_list.append(lc_corr.to_periodogram(**kwargs))
+                    except IndexError:
+                        pixel_list.append(None)
+                else:
+                    if normalize:
+                        if len(lc_corr.remove_nans().flux) == 0:
+                            pixel_list.append(None)
+                        else:
+                            pixel_list.append(lc_corr)
+                    elif not normalize:
+                        if len(lc.remove_nans().flux) == 0:
+                            pixel_list.append(None)
+                        else:
+                            pixel_list.append(lc)
+
+        mask = self._parse_aperture_mask(aperture_mask)
+        tpf_plot = plt.imshow(self.flux[0])
+        plt.close()
+
+        with plt.style.context(style):
+            if title is None:
+                title = f'Target ID: {self.targetid}'
+
+            fig = plt.figure()
+            if ax == None:
+                ax = plt.gca()
+                ax.get_xaxis().set_ticks([])
+                ax.get_yaxis().set_ticks([])
+
+                if periodogram:
+                    ax.set(title=title, xlabel='Frequency', ylabel='Power')
+                else:
+                    if normalize:
+                        ax.set(title=title, xlabel='Time', ylabel='Normalized Flux')
+                    else:
+                        ax.set(title=title, xlabel='Time', ylabel='Flux')
+
+            gs = gridspec.GridSpec(self.shape[1], self.shape[2], wspace=0.01, hspace=0.01)
+
+            for k in range(self.shape[1]*self.shape[2]):
+                if pixel_list[k]:
+                    x, y = np.unravel_index(k, (self.shape[1], self.shape[2]))
+
+                    if periodogram:
+                        x_vals = pixel_list[k].frequency.value
+                        y_vals = pixel_list[k].power.value
+                        x_axis_min = 0
+                        x_axis_max = np.nanmax(x_vals)
+                        y_axis_min = 0
+                        y_axis_max = max(np.nanmax(y_vals),1e-99)
+                    elif not periodogram:
```

Comment:
Nitpick: this can of course just be `else`:
```suggestion
                    else:
```